btusb.c

```diff
@@ -28,7 +28,7 @@
 #include "btrtl.h"
 #include "btmtk.h"
 
-#define VERSION "0.8"
+#define VERSION "0.81"
```

Comment:
As discussed in the overall commentary - I had to change these VERSIONs or DKMS refuses to install them as they match ubuntu's upstream version.  Not sure if there's some alternative standard for versions to add a separate epoch from upstream so that upstream changes beats this one, but this one beats current kernels,...